_posts/2024-04-25-WildFly32-Released.adoc

```diff
@@ -146,17 +146,17 @@ The following table lists the various Jakarta EE technologies offered by WildFly
 | Jakarta MVC
 (_preview stability only_)| 2.1| N/A xref:note2[^2^]
 
+|Jakarta Pages| 3.1 |10
```

Comment:
@scottmarlow BTW this seems to me to be a tiny bit premature, the spec is still called "Jakarta Server Pages" in the version 3.1 - the rename happens with 4.0 as per https://jakarta.ee/specifications/pages/ and even https://projects.eclipse.org/projects/ee4j.jsp still lists this under old name.

Reply:
Good catch that there is some confusing information as you quoted @rhusar!  Should we include more detail about the change somewhere?  

The top of https://jakarta.ee/specifications/pages/ is reflective of the new name I think and eventually the old name will not appear before after a few more Jakarta Pages releases.

Reply:
Sigh.

Thank you for pointing this out, @rhusar. At this point I just want to leave it as is though; I don't have time for further tweaking.

Re adding more detail about the change somewhere, I think people will figure it out. If we stick it in our docs it will just become cruft.

Reply:
Agreed, lets just leave it as is and thus help promote the new name going forward.